src/main/scala/org/openeo/opensearch/backends/STACClient.scala

```diff
@@ -53,6 +53,7 @@ class STACClient(private val endpoint: URL=new URL("https://earth-search.aws.ele
       .param("limit", "100")
       .param("bbox", bboxParam)
       .param("page", page.toString)
+      .timeout(connTimeoutMs = 10000, readTimeoutMs = 40000)
```

Comment:
This is already covered by the change in OpenSearchClient.

Reply:
fixed